dockstore_launcher_config/compose.config

```diff
@@ -1,11 +1,10 @@
 {
 "PUBLIC_LAUNCHER_IP_ADDRESS":"",
+"PRODUCTION":false,
 "DOCKSTORE_VERSION":"1.8.0-beta.4",
-"GITHUB_CLIENT1_ID":"foobared",
+"UI2_HASH":"develop",
```

Comment:
doesn't seem like an obvious improvement, isn't this lacking the hash anyway?
(i.e. close enough to reality, but not close enough?)